### DISCUSSION TITLE
Update of a few lines that weren't translated yet in FR

### DISCUSSION THREAD

**Discussion by booToToM** [aclap-dev/video-downloadhelper#967] [Language & Translation]
[VDH Translate TTM 20240704 fr.txt](https://github.com/user-attachments/files/16099492/VDH.Translate.TTM.20240704.fr.txt)

Here are several lines translated.

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
